src/lib/fetch.ts

```diff
@@ -211,24 +229,14 @@ async function fetchAccountInternal(
 }
 
 type FetchConfig = { timeout?: number };
-type FetchResponse = { data: any; errors?: any };
+type FetchResponse<TDataResponse = any> = { data: TDataResponse; errors?: any };
```

Comment:
We add a generic type here for `FetchResponse` to get better type safety by passing in types on requests we make.